apiaudio/api_request.py

```diff
@@ -8,8 +8,12 @@
 
 class APIRequest:
     def _api_key_checker(api_key=None):
-        if api_key == None or api_key == "your-key":
+        PLACEHOLDERS = ["your-key", "APIKEY", "API_KEY"]
+        if api_key == None or api_key in PLACEHOLDERS:
```

Comment:
we can have `api_key = os.environ.get("apiaudio_key", os.environ.get("aflr_key", None))` here, I think the double if check for `api_key == None` can be avoided this way

Reply:
Added. Thanks!